components/config/opts.rs

```diff
@@ -413,7 +413,7 @@ pub fn default_opts() -> Opts {
         unminify_js: false,
         local_script_source: None,
         print_pwm: false,
-        minibrowser: false,
+        without_minibrowser: false,
```

Comment:
can this be `minibrowser: true,` to avoid the double negative? then on line 771, we can invert the result of the opt_present call

Reply:
hmm, done in [f23d5f6](https://github.com/servo/servo/pull/30338/commits/f23d5f6c550eb2ff1590bcd5a385c769762e3608) but we couldn't escape double negative completely! See browser.rs.

Reply:
we kept the original name `minibrowser` in [4f70fb5](https://github.com/servo/servo/pull/30338/commits/4f70fb5e338edbe2cae80cc7d5b66370b73ec483).  With other things, it also helped us avoid double negative